packages/email-petition/src/email-petition.tsx

```diff
@@ -256,6 +313,58 @@ export function EmailPetition(props: {
 
   const recaptchaRef = useRef<ReCAPTCHA>(null);
 
+  // Function to call zipToCityLookup API
+  const fetchCityByZip = async (zipCode: string) => {
+    // Don't make API calls if Google Maps API is disabled
+    if (props.citySelectionMode !== "autocompleteTextbox") {
+      return;
+    }
```

Comment:
pls remove. it's out of scope for this function and should be checked by the calling function, which it already is.